composer.json

```diff
@@ -6,7 +6,8 @@
     "homepage": "http://developers.google.com/api-client-library/php",
     "license": "Apache-2.0",
     "require": {
-        "php": ">=5.2.1"
+        "php": ">=5.4",
+        "google/auth": "dev-master"
```

Comment:
once this branch is merged into `master`, we should tag the version of `google/auth` using [semantic versioning](http://semver.org/)